src/ape/types/trace.py

```diff
@@ -25,6 +25,31 @@
 """
 
 
+class CoverageItem(BaseModel):
```

Comment:
This represents a relatively complex concept, can it be explained more what this item represents? in terms of "node" and "location"

Does it link to an AST expression? Statement? Programmatic block?

Reply:
- [ ] Doc more
- [ ] Increase granularity to include columns (temporarily ignoring because of some complication, i forgot about that)

Reply:
right now if 2 source-based nodes have the exact same location, it only counts as 1. however if the columns differ, it should count as more than 1.

what else... the implicit checks a statement. Right now, it is only 1 "statement" per implicit check because that is how the error mapping working but we could add more if needed and if it possible.